src/groups/meta_group_wrapper.cpp

```diff
@@ -388,26 +450,28 @@ Napi::Value MetaGroupWrapper::infoDestroy(const Napi::CallbackInfo& info) {
 
 Napi::Value MetaGroupWrapper::memberGetAll(const Napi::CallbackInfo& info) {
     return wrapResult(info, [&] {
-        std::vector<session::config::groups::member> allMembers;
+        std::vector<Napi::Object> allMembersJs;
```

Comment:
Why do we lose member type for a general object here?

Reply:
Ah what I think I got it. The memberGetAll is only returning a subset of the member type which includes the status? I guess for convenience?

Reply:
yep, we need to add the status to an object that doesn't have it